src/main/java/org/isf/operation/manager/OperationBrowserManager.java

```diff
@@ -144,11 +144,11 @@ public boolean descriptionControl(String description, String typeCode) throws OH
 	 * Get the list of possible operation results
 	 * @return the found list
 	 */
-	public ArrayList<String> getResultsList() { //TODO (OP-372): deal with codes instead of translations!!!
+	public ArrayList<String> getResultsList() { 
 		ArrayList<String> resultsList = new ArrayList<String>();
-		resultsList.add(MessageBundle.getMessage("angal.operation.result.sucess"));
-		resultsList.add(MessageBundle.getMessage("angal.operation.result.failure"));
-		resultsList.add(MessageBundle.getMessage("angal.operation.result.undefined"));
+		resultsList.add("angal.operation.result.sucess");
+		resultsList.add("angal.operation.result.failure");
+		resultsList.add("angal.operation.result.undefined");
```

Comment:
fix: sucess -> success

Anyway, with this method, in the DB will finish the whole string "angal.operation.result.sucess":

![image](https://user-images.githubusercontent.com/2938553/104066191-4621d780-5201-11eb-95cb-2de0d8ab4ef9.png)

What about using a mapping mechanism as done in (e.g.) [ExaminationBrowserManager.java:92](https://github.com/informatici/openhospital-core/blob/develop/src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java#L92) ?

Reply:
> What about using a mapping mechanism as done in (e.g.) [ExaminationBrowserManager.java:92](https://github.com/informatici/openhospital-core/blob/develop/src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java#L92) ?

It was just a choice, I can do it like that if necessary.

Reply:
I think it's better as pattern; also step_57_multiple_operations_in_admission.sql should be changed after.
Thanks!

Reply:
Hi @mwithi, I made a new commit that uses the mapping mechanism as requested. and for what needs to be saved in bd, I used the keys already present in step_57_multiple_operations_in_admission.sql file and I think it won't be necessary to update it anymore.